build.gradle

```diff
@@ -76,6 +76,7 @@ dependencies {
     deobfCompile "com.feed_the_beast.mods:FTBLib:${ftblib_version}"
     deobfCompile "infinitylib:infinitylib:${infinitylib_version}"
     deobfCompile "agricraft:AgriCraft${agricraft_version}"
+    compile fileTree(include: ['*.jar'], dir: 'libraries')

```

Comment:
added this because eb was in this folder (\libraries\ElectroblobsWizardry-4.2.8-MC1.12.2.jar), but there are probably better ways to include it with gradle?